scripts/setup/ubuntu/install_deps.sh

```diff
@@ -26,6 +26,7 @@ DEPS=(
   wget
   zlib1g
   zlib1g-dev
+  jq
```

Comment:
?

Reply:
> @celinval
> 
> > 1. Is there a reason why you preferred `.cargo/config.toml` instead of `clippy.toml`.
> 
> I don't think that is possible, and keeping it so seems to be the stance from clippy devs: [rust-lang/rust-clippy#1313 (comment)](https://github.com/rust-lang/rust-clippy/issues/1313#issuecomment-633991593). The idea behind it seems to be that you would control all lints at once, not just clippy lints.
> 
> > 2. How long does `cargo clippy` takes? I was wondering if we should enable this in our regression.
> 
> It takes no longer than 2 compiles with `cargo build --workspace`. This is the first job to finish.

This is so confusing... That's totally what they recommend in their documentation: https://github.com/rust-lang/rust-clippy#configuration

Reply:
@tedinski `jq` is required for generating the warning stats given at the bottom of .github/workflows/format-check.yml

Reply:
@celinval Yea, both are under the same section, which is not great.

I could not find a single example of allowing lints with `clippy.toml` so I went looking around and landed on that issue thread.

Reply:
Closing as this appears to be a question rather than a request.

Reply:
I would put instally jq into that workflow, not as a general dependency.

Reply:
Moved in commit 455c42f492c79745e94ceaed92acf8cc0d27ed14